lib/ex_unit/lib/ex_unit.ex

```diff
@@ -289,6 +294,14 @@ defmodule ExUnit do
     Keyword.put(opts, :max_cases, max_cases(opts))
   end
 
+  defp put_slowest(opts) do
```

Comment:
Since we are changing the value of trace based on other options, we need to add it to the list here: https://github.com/sorentwo/elixir/blob/5cec250feb2c7ae89aa022a62ca1a6a94a3ad7b7/lib/ex_unit/lib/ex_unit.ex#L283